packages/transformers/stylus/src/StylusTransformer.js

```diff
@@ -46,7 +54,7 @@ export default (new Transformer({
     ): Stylus);
 
     let code = await asset.getCode();
-    let style = stylus(code, stylusConfig);
+    let style = stylus(code, {...stylusConfig});
```

Comment:
Does stylus really mutate the config object?

Reply:
Yes 😞